### ISSUE TITLE
I cannot install libgnomeui.

### ISSUE THREAD

**Issue by crocket** [aurapm/aura#445]
```
~> sudo aura -A libgnomeui
[sudo] password for xxx: 
aura >>= Determining dependencies...
aura >>= Dependency checking failed for these reasons:
The dependency `libgnome-data` could not be found. You may need to search for a package to satisfy it.
```

Yaourt can install libgnomeui

**Comment by fosskers** [comment]
This is Aura having trouble with virtual packages. I've had problems with `gnome-data` myself in the past, so this is a known issue.

**Comment by fosskers** [comment]
This seems fixed on `master`, which will be released soon as Aura 2.0.
